projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -291,12 +295,10 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 			) }
 
 			{ contentIsLoaded && (
-				<ShowLittleByLittle
-					showAnimation={ ! attributes.animationDone }
-					onAnimationDone={ () => {
-						setAttributes( { animationDone: true } );
-					} }
+				<ShowContent
 					html={ attributes.content }
+					setAttributes={ setAttributes }
```

Comment:
I think that is a very bad way of doing it, you should rely on a more "React" way and create a "onChange" (and reinstate "onAnimationDone"). This way the component something passed by the parent.

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/28575#:~:text=main%20render%20fn-,2b770d6,-romarioraffington%20dismissed%20n3f